python/ray/serve/api.py

```diff
@@ -571,3 +577,153 @@ def delete(name: str, _blocking: bool = True):
     """
     client = get_global_client()
     client.delete_apps([name], blocking=_blocking)
+
+
+@PublicAPI(stability="alpha")
+def multiplexed(
+    func: Optional[Callable[..., Any]] = None, max_num_models_per_replica: int = -1
+):
+    """[EXPERIMENTAL] Coverts a function or method to a multiplexed function.
+
+    The function can be standalone function or a method of a class. The
+    function must have exact one argument with type `str` and the argument
+    must be the first argument. The `str` argument is the model id of the model to
+    be loaded.
```

Comment:
This is a bit confusing to me. Why mention it must be the first argument if it can only take one argument? Lmk if I'm interpreting this incorrectly.
```suggestion
    function must have exactly one argument, the model id of type `str` for the
    model to be loaded.
```